Simulator/Core/Driver.cpp

```diff
@@ -5,7 +5,7 @@
  *
```

Comment:
Let's change this class to `Core`.

Reply:
Modified the filename `Driver.cpp` and `Driver.h` to `Core.cpp` and `Core.h` respectively.